host/sgx/create.c

```diff
@@ -608,6 +619,20 @@ oe_result_t oe_sgx_validate_enclave_properties(
         goto done;
     }
 
+    if (properties->config.flags.create_zero_base_enclave)
+    {
+        if (!oe_sgx_is_valid_start_address(properties->config.start_address))
```

Comment:
If this only checks if the start_address is page-aligned, you can just do

if (properties->config.start_address & OE_PAGE_SIZE)

then you can get rid of oe_sgx_is_valid_start_address functions and the MASK define.

Reply:
The function oe_sgx_is_valid_start_address is needed in tools/oesign/oesign.c too. I have removed the mask define, using % instead to enforce page alignment.